share/completions/climate.fish

```diff
@@ -0,0 +1,82 @@
+# fish completion for climate
+
+function __fish_is_git_repository
+    git rev-parse --is-inside-work-tree ^/dev/null >/dev/null
+    return $status
+end
+
+function __fish_climate_already_has_argument
```

Comment:
I'm a bit confused by what this function is supposed to do given its name.

`__fish_climate_already_has_argument` implies that it should return 0 (true) if the command line has an argument but it clearly does the exact opposite. Have I missed something?